src/test/java/io/pinecone/PineconeBuilderTest.java

```diff
@@ -106,6 +106,6 @@ public void PineconeWithSourceTag()  throws IOException {
 
         assertEquals(expectedIndex, index);
         verify(mockClient, times(1)).newCall(requestCaptor.capture());
-        assertEquals("lang=java; pineconeClientVersion=" + pineconeClientVersion + "; source_tag=testsourcetag", requestCaptor.getValue().header("User-Agent"));
+        assertEquals("lang=java; pineconeClientVersion=" + pineconeClientVersion + "; source_tag=testSourceTag", requestCaptor.getValue().header("User-Agent"));
```

Comment:
Not related to proxy but for some reason, the `testsourcetag` value was accidentally set to lowercase instead of camelcase.